CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ All notable, unreleased changes to this project will be documented in this file.
 # 3.12.0 [Unreleased]
 
 ### Breaking changes
+- `stocks` and `channelListings` inputs in `ProductVariantBulkUpdate` mutation has been changed.
```

Comment:
Can we add what has changed and mention that it was a preview API that has changed.